### PR TITLE
Fixed "Search"

### PR DESCRIPTION
**Pull request by YuliaFursovych**

### CODE REVIEW

**Review by Blager89 — APPROVED** [review]

File: client/src/components/AppHeader/AppHeader.js [hunk 1]
```diff
@@ -1,48 +1,42 @@
...
-import { Navbar,  FormControl, Form, Nav, InputGroup, Button } from 'react-bootstrap';
+import './AppHeader.scss'
+import { Navbar,  FormControl, Form, Nav, InputGroup } from 'react-bootstrap';
 import { faSignInAlt } from '@fortawesome/free-solid-svg-icons'
 import { faUsers } from '@fortawesome/free-solid-svg-icons'
 import { faCalendarAlt } from '@fortawesome/free-solid-svg-icons'
-import { faSearch } from '@fortawesome/free-solid-svg-icons'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 
 const AppHeader = props => {
```
Comment (yodeco) [inline client/src/components/AppHeader/AppHeader.js]: You can shorten this syntax const AppHeader = props => {return ()} to more simple variant
AppHeader = props => ()

File: client/src/components/AppHeader/AppHeader.js [hunk 2]
```diff
@@ -1,48 +1,42 @@
 import logo from './logo.png';
 import React from 'react';
-import './style.scss'
-import { Navbar,  FormControl, Form, Nav, InputGroup, Button } from 'react-bootstrap';
+import './AppHeader.scss'
```
Comment (yodeco) [inline client/src/components/AppHeader/AppHeader.js]: ;

File: client/src/components/AppHeader/AppHeader.scss [hunk 3]
```diff
@@ -51,31 +51,20 @@
...
-            width: 35px;
+            width: 75px;
             transition: width 0.5s;
             border: 0;
             outline: none;
-            border-bottom-right-radius: 50px;
-            border-top-right-radius: 50px;
+            border-radius: 50px;
             box-shadow: none;
+            text-decoration: #e9ece5;
```
Comment (yodeco) [inline client/src/components/AppHeader/AppHeader.scss]: this property gets coverted to 
    text-decoration-line: initial;
    text-decoration-style: initial;
    text-decoration-color: #e9ece5;

Maybe as variant better just point   text-decoration-color: #e9ece5; ? Win is 1 css rule, not 3.

File: client/src/components/AppHeader/AppHeader.js (hunk 2 above)
Comment (YuliaFursovych) [reply client/src/components/AppHeader/AppHeader.js]: Yes, I will add. Thank you for seeing.

File: client/src/components/AppHeader/AppHeader.js (hunk 1 above)
Comment (IgorPopach) [reply client/src/components/AppHeader/AppHeader.js]: Later we'll create state in this component, so I think we can pretermit this...

Comment (YuliaFursovych) [reply client/src/components/AppHeader/AppHeader.js]: Okay, thank you!

File: client/src/components/AppHeader/AppHeader.scss (hunk 3 above)
Comment (YuliaFursovych) [reply client/src/components/AppHeader/AppHeader.scss]: Thank you, I will fix.

Comment (YuliaFursovych) [reply client/src/components/AppHeader/AppHeader.scss]: Made a new commit with amendments.

**Review by yodeco — APPROVED** [review]

**Pull request merged**